matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/livelocation/LiveLocationShareAggregatedSummary.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.room.model.livelocation
+
+import org.matrix.android.sdk.api.session.room.model.message.MessageBeaconLocationDataContent
+
+/**
+ * Aggregation info concerning a live location share.
+ */
+data class LiveLocationShareAggregatedSummary(
+        /**
+         * Event id of the event that started the live.
+         */
+        val eventId: String,
+        val roomId: String,
```

Comment:
Do we really need to have the `roomId` here?

Reply:
The reason I added the `roomId` is because I think we may need it at some point. I guess we may need to query all current lives running for a given room for example. Or get the room ids corresponding to the current running lives.

Reply:
OK for the DB, I see your point. But for `LiveLocationShareAggregatedSummary`, which is used in `EventAnnotationsSummary` I think the `roomId` can be removed (and not mapped from the DB).

Reply:
`LiveLocationShareAggregatedSummaryMapper.map(LiveLocationShareAggregatedSummary)` can be removed, it's not used.

The mappers of this type is to map the db content to the model the SDK exposes to the app. So it's OK that the DB contains more info (for queries, etc.), and only a part of that is mapped and pass to the app.

Reply:
Yes you are right, I removed non necessary fields: [1720dc1](https://github.com/vector-im/element-android/pull/5869/commits/1720dc1fac27e9f3602469c3085ae603712c15a5)

Reply:
I also did it for the other 2 cases: https://github.com/vector-im/element-android/pull/5890 to be coherent.